components/record/MediaImage.vue

```diff
@@ -6,11 +6,13 @@
     <img
       :src="src"
       class="mw-100 mb-3"
+      :alt="link"
```

Comment:
This sets the alt text to the edm:isShownAt (via the redirection API method)... definitely not good alternate text for an image. Either leave it blank or set it to something like "View this object at the provider's website"